src/hooks/custom/useInsightInput.ts

```diff
@@ -0,0 +1,54 @@
+import { InsightPostRequest } from '@/types/insight';
+import { useCheckMainImage } from '@/utils/upload';
+import { useState } from 'react';
+import defaultImage from '@image/defaultImage.jpeg';
+import { ParsedUrlQuery } from 'querystring';
+
+interface InsightSummary {
+  title: string | undefined;
+  summary: string | undefined;
+  keywords: string | undefined;
+  folderName: string[] | undefined;
```

Comment:
```suggestion
  title?: string;
  summary?: string;
  keywords?: string;
  folderName?: string[];
```

위와 같이 바꾸면 더 간결할 것 같습니다.